pkg/progress/progress.go

```diff
@@ -201,39 +204,40 @@ func (m *Manager) GetProgresses(filter func(p string) bool) []string {
 	m.RLock()
 	defer m.RUnlock()
 
-	processes := []string{}
+	progresses := []string{}
```

Comment:
```suggestion
	progresses := make([]string, 0, len(m.progresses))
```

Reply:
fixed in https://github.com/tikv/pd/pull/8276/commits/c1071298712d7dd3fecbec8e3b674ba5e6b02a8c